pkg/aws/ec2/ec2.go

```diff
@@ -297,20 +280,24 @@ func (c *Client) GetVpcs(ctx context.Context) (ipamTypes.VirtualNetworkMap, erro
 
 // describeSubnets lists all subnets
 func (c *Client) describeSubnets(ctx context.Context) ([]ec2_types.Subnet, error) {
-	c.limiter.Limit(ctx, "DescribeSubnets")
-
-	sinceStart := spanstat.Start()
+	result := []ec2_types.Subnet{}
```

Comment:
same here for the initialization of the slice.

Reply:
Updated